runtime/lua/vim/lsp.lua

```diff
@@ -293,6 +293,19 @@ local function text_document_did_open_handler(bufnr, client)
   }
   client.notify('textDocument/didOpen', params)
   util.buf_versions[bufnr] = params.textDocument.version
+
+  -- Next chance we get, we should re-do the diagnostics
+  vim.schedule(function()
+    vim.lsp.handlers["textDocument/publishDiagnostics"](
```

Comment:
Just curious,  is there a reason this isn't calling the display function of the diagnostic module?
Would avoid running through the save logic again

Reply:
This re-uses the users configuration and/or any handlers that they have saved to the default handler.

Reply:
(otherwise, if you had virtual text disabled, it would flash the virtual text on load and then take it off next time you got diagnostics)

Reply:
Ah, alright. Thank you for the clarification